frontend/.env.example

```
@@ -0,0 +1,10 @@
+# Frontend Environment Variables
+
+# Backend API URL
+VITE_BACKEND_URL=http://localhost:5001
+
+# ZegoCloud Configuration
+VITE_ZEGO_APP_ID=your_zego_app_id
+
+# Optional (not recommended for production exposure)
+VITE_ZEGO_SERVER_SECRET=your_zego_server_secret
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)